flows/common/common.go

```diff
@@ -208,34 +280,36 @@ func Rate(flow *otg.Flow, hdrs []gopacket.SerializableLayer) (uint64, error) {
 	return pps, nil
 }
 
+// flowPackets returns the number of packets that should be sent for a
+// particular flow. If the specification is not provided it returns 0,
```

Comment:
ah I see, you have the behavior documented here.

Reply:
Added a clarifying comment in the loop too.